config/config.yaml.example

```diff
@@ -47,7 +47,8 @@ samples:
       colour: "#3DB7E9"
 
 # Resource specifications for different steps
```

Comment:
Lines 51 to 55 should go after line 16. (Comment stays with the rule resources).